.gitignore

```diff
@@ -12,6 +12,8 @@
 *.so
 *.dll
 *.dylib
+# ignore 'so's with appended version: libqmlbind.so.1.0.0, libqmlbindso.1.0 etc.
+libqmlbind.so.*
```

Comment:
To catch both cases, we'd need `*.so*`, which is quite generic. I hesitated to use that because I was unsure to exclude every file with “.so“ somewhere in the name, i.e. at the start or somewhere in the middle. The intention of `libqmlbind.so.*` (or `libqmlbind.so*`) was to make it more specific.

Would you prefer `*.so*` or `libqmlbind.so*`?